Schema.js

```diff
@@ -60,6 +60,9 @@ defaultColumns = {
     "expiresAt":      {type:'Date'},
     "createdWith":    {type:'Object'},
   },
+  _GlobalConfig: {
```

Comment:
This `defaultColumns` object only tracks default columns for collections that contain Parse Objects (thats what `_SCHEMA` isn't in here. So I think config probably shouldn't be here either.